R/plumber-step.R

```diff
@@ -196,6 +196,8 @@ PlumberEndpoint <- R6Class(
     },
     #' @field params endpoint parameters
     params = NA,
+    #' @field serializer_params serializer parameters
+    serializer_params = NULL,
```

Comment:
As much as I don't like it, let's make the camelCase and change to `dynamicSerializerParams` (or something to hint that it is the _dynamic_ serializer params... `dynSerParams`?)

Will need to change all usage as well. :-/